packages/echo-core/rollup.config.js

```diff
@@ -20,7 +20,7 @@ export default [
         plugins: [
             tslibResolveId(),
             del({ targets: 'dist/*', runOnce: true }),
-            ts2(),
+            typescript({ outDir: './dist/' }),
```

Comment:
After changing to the official rollup typescript plugin (`@rollup/plugin-typescript`) it pointed out some missing settings here, based on the typescript config.

In this particular example: in the ts config `"declaration": true,` is set, so we need to specify here an output dir.